examples/react-native/README.md

```diff
@@ -0,0 +1,33 @@
+# Lens SDK + react-native
+
+This package is an example app showcasing the `@lens-protocol/react` integration with [react-native](https://reactnative.dev/).
+
+The example app is using a wonderful [`react-native-mmkv`](https://github.com/mrousavy/react-native-mmkv) as a storage provider and very basic wallet implementation.
+
+It is not published or released anywhere directly.
+
+## Getting Started
+
+Install example dependencies via (should be run from the `examples/react-native` folder:
+```bash
+pnpm install
+```
+
+Make sure your development environment is ready to start locally `react-native` app by following the [official docs](https://reactnative.dev/docs/environment-setup).
```

Comment:
May be worthwhile to include some detail around the basic RN setup here. 

I had some issues relating to the fact I didn't have pods installed

Reply:
fixed

Reply:
The readme is good, let's release it and wait for feedback, then we will see what are the most common issues and we will improve the readme based on that